features/lighthouse-visualizations/documentation/README.md

```diff
@@ -1,17 +1,15 @@
 ## Lighthouse Score Visualizations (Labs feature)
 
-<img width="1400" alt="Deploy view with Lighthouse visualizations" src="https://user-images.githubusercontent.com/79875905/160019039-c3e529de-f389-42bc-a3d4-458c90d59e6a.png">
-
-If you have the Lighthouse Build Plugin installed, you can now view the Lighthouse scores for your builds on your site's Deploy page by enabling the Lighthouse Visualizations feature under Labs. 
+When you install the [Lighthouse Build Plugin](https://app.netlify.com/teams/netlify/plugins/@netlify/plugin-lighthouse/install) on your site and enable this experimental feature, you can view the Lighthouse scores for each of your builds on your site's Deploy Details page with a much richer format.
 
-<img width="1400" alt="Labs UI" src="https://user-images.githubusercontent.com/79875905/160018999-0fde5619-15c9-4622-99a2-6ec64fc69826.png">
+<img width="1400" alt="Deploy view with Lighthouse visualizations" src="https://user-images.githubusercontent.com/79875905/160019039-c3e529de-f389-42bc-a3d4-458c90d59e6a.png">
 
 If you have multiple audits (directories, paths, etc) defined in your build, we will display a roll-up of the average Lighthouse scores for all the current build's audits plus the results for each individual audit.
 
 <img width="1400" alt="Deploy details with multiple audit Lighthouse results" src="https://user-images.githubusercontent.com/79875905/160019057-d29dffab-49f3-4fbf-a1ac-1f314e0cd837.png">
 
 Some items of note:
-- The Lighthouse Build Plugin must be installed on your site(s) in order for these score visualizations to be displayed.
+- The [Lighthouse Build Plugin](https://app.netlify.com/teams/netlify/plugins/@netlify/plugin-lighthouse/install) must be installed on your site(s) in order for these score visualizations to be displayed.
```

Comment:
You should remove /teams/netlify from the URL.

Reply:
👍